### DISCUSSION TITLE
Let's Propose Ways to Organize the Math AutoCorrect Codes to Improve the Documentation

### DISCUSSION THREAD

**Discussion by brichwin** [daisy/math-a11y#35] [General]
Note: The current rendering of the recommendation is here: [List of Commonly Used Math AutoCorrect Entries in Word 365](https://daisy.github.io/math-a11y/docs/ms-math/Listv2-of-Commonly-Used-Math-AutoCorrect-Entries-in-Word-365.html)

One of the areas of improvement listed in the [Microsoft support page needs improvement (#12)](https://github.com/daisy/math-a11y/issues/12) is that the Math AutoCorrect codes table could be more usefully organized to help find the code you are looking for.

Let's use this discussion space to discuss how they can be organized and then create one or more proposed listings on the [Suggested Math AutoCorrect Code Listings Organization](https://github.com/daisy/math-a11y/wiki/Suggested-Organization-of-Math-AutoCorrect-Entries) wiki page.

**Comment by brichwin** [comment]
I added the script that I made for generating SVG graphic files from mathematics symbols to the repo in the tools folder: [stix_two_math_chars_to_svg](https://github.com/daisy/math-a11y/tree/main/tools/stix_two_math_chars_to_svg)

**Comment by ejpyattpsu** [comment]
FYI - I wrote a blog post a while ago about using [combining diacritics to write x̄ and p̂](https://sites.psu.edu/gotunicode/2010/03/26/dealing_with_x-bar_x_and_p-hat/ ). Combining diacritics are useful for locations that don't fully support MathML (like Facebook or plain text email).

Other information on combining diacritics and math symbols outside of MathML can be found on the two sites below.

-[Combining Diacritics in Phonetics](https://sites.psu.edu/symbolcodes/ipavowels/)
-[Math Symbol Reference (Partial)](https://sites.psu.edu/symbolcodes/accents/math/mathchart/)

More symbols can be found in CharacterMap (Windows) or the Emoji & Symbols Palette (Mac).

Hope this helps.
Elizabeth
